apps/vr-tests-react-components/src/stories/Avatar.stories.tsx

```diff
@@ -2,6 +2,7 @@ import { storiesOf } from '@storybook/react';
 import * as React from 'react';
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 74 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27572&commitId=582a326962e47176ea0f58b940d6149cee176cf8&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1859 screenshots unchanged, 0 screenshots with different dimensions and 72 screenshots with visible difference.


  <details><summary><b>unknown</b> 74 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.basic - Dark Mode.normal.chromium.png|9| Changed|
|Avatar Converged.basic - High Contrast.normal.chromium.png|13| Changed|
|Avatar Converged.basic - RTL.normal.chromium.png|14| Changed|
|Avatar Converged.basic.normal.chromium.png|14| Changed|
|Avatar Converged.customSize+icon+active.normal.chromium.png|700| Changed|
|Avatar Converged.customSize+name+badge.normal.chromium.png|10| Changed|
|Avatar Converged.size+image+badge.normal.chromium.png|182| Changed|
|Avatar Converged.size+inactive+badge.normal.chromium.png|119| Changed|
|AvatarGroup Converged.layoutStack - High Contrast.default.chromium.png|7| Changed|
|Persona Converged.basic - High Contrast.normal.chromium.png|6| Changed|
|Table layout flex - cell actions.always visible - Dark Mode.hover row.chromium.png|1| Changed|
|Table layout flex - cell actions.always visible - High Contrast.chromium.png|25| Changed|
|Table layout flex - cell actions.always visible - High Contrast.hover row.chromium.png|51| Changed|
|Table layout flex - cell actions.default - Dark Mode.hover row.chromium.png|1| Changed|
|Table layout flex - cell actions.default - High Contrast.chromium.png|25| Changed|
|Table layout flex - cell actions.default - High Contrast.hover row.chromium.png|51| Changed|
|Table layout flex - headers.sortable - High Contrast.chromium.png|25| Changed|
|Table layout flex - headers.sortable - High Contrast.hover header.chromium.png|25| Changed|
|Table layout flex - headers.sortable - High Contrast.press header.chromium.png|25| Changed|
|Table layout flex.multiselect (checked) - brand - Dark Mode.chromium.png|3| Changed|
|Table layout flex.multiselect (checked) - brand - High Contrast.chromium.png|201| Changed|
|Table layout flex.multiselect (checked) - brand - RTL.chromium.png|3| Changed|
|Table layout flex.multiselect (checked) - brand.chromium.png|3| Changed|
|Table layout flex.multiselect (checked) - neutral - High Contrast.chromium.png|128| Changed|
|Table layout flex.multiselect (checked) - neutral - RTL.chromium.png|7| Changed|
|Table layout flex.multiselect (checked) - neutral.chromium.png|7| Changed|
|Table layout flex.multiselect (mixed) - brand - Dark Mode.chromium.png|2| Changed|
|Table layout flex.multiselect (mixed) - brand - High Contrast.chromium.png|113| Changed|
|Table layout flex.multiselect (mixed) - brand - RTL.chromium.png|1| Changed|
|Table layout flex.multiselect (mixed) - brand.chromium.png|1| Changed|
|Table layout flex.multiselect (mixed) - neutral - High Contrast.chromium.png|77| Changed|
|Table layout flex.multiselect (mixed) - neutral - RTL.chromium.png|4| Changed|
|Table layout flex.multiselect (mixed) - neutral.chromium.png|4| Changed|
|Table layout flex.multiselect - High Contrast.chromium.png|25| Changed|
|Table layout flex.single select (checked) - High Contrast.chromium.png|25| Changed|
|Table layout flex.single select (checked) - brand - High Contrast.chromium.png|69| Changed|
|Table layout flex.single select (checked) - neutral - High Contrast.chromium.png|51| Changed|
|Table layout flex.single select (checked) - neutral - RTL.chromium.png|2| Changed|
|Table layout flex.single select (checked) - neutral.chromium.png|2| Changed|
|Table layout flex.single select - High Contrast.chromium.png|25| Changed|
|Table layout flex.size - medium - High Contrast.chromium.png|25| Changed|
|Table layout table - cell actions.always visible - Dark Mode.hover row.chromium.png|1| Changed|
|Table layout table - cell actions.always visible - High Contrast.chromium.png|25| Changed|
|Table layout table - cell actions.always visible - High Contrast.hover row.chromium.png|51| Changed|
|Table layout table - cell actions.default - Dark Mode.hover row.chromium.png|1| Changed|
|Table layout table - cell actions.default - High Contrast.chromium.png|25| Changed|
|Table layout table - cell actions.default - High Contrast.hover row.chromium.png|51| Changed|
|Table layout table - headers.sortable - High Contrast.chromium.png|25| Changed|
|Table layout table - headers.sortable - High Contrast.hover header.chromium.png|25| Changed|
|Table layout table - headers.sortable - High Contrast.press header.chromium.png|25| Changed|
|Table layout table.multiselect (checked) - brand - Dark Mode.chromium.png|3| Changed|
|Table layout table.multiselect (checked) - brand - High Contrast.chromium.png|201| Changed|
|Table layout table.multiselect (checked) - brand - RTL.chromium.png|3| Changed|
|Table layout table.multiselect (checked) - brand.chromium.png|3| Changed|
|Table layout table.multiselect (checked) - neutral - High Contrast.chromium.png|128| Changed|
|Table layout table.multiselect (checked) - neutral - RTL.chromium.png|7| Changed|
|Table layout table.multiselect (checked) - neutral.chromium.png|7| Changed|
|Table layout table.multiselect (mixed) - brand - Dark Mode.chromium.png|2| Changed|
|Table layout table.multiselect (mixed) - brand - High Contrast.chromium.png|113| Changed|
|Table layout table.multiselect (mixed) - brand - RTL.chromium.png|1| Changed|
|Table layout table.multiselect (mixed) - brand.chromium.png|1| Changed|
|Table layout table.multiselect (mixed) - neutral - High Contrast.chromium.png|77| Changed|
|Table layout table.multiselect (mixed) - neutral - RTL.chromium.png|4| Changed|
|Table layout table.multiselect (mixed) - neutral.chromium.png|4| Changed|
|Table layout table.multiselect - High Contrast.chromium.png|25| Changed|
|Table layout table.single select (checked) - High Contrast.chromium.png|25| Changed|
|Table layout table.single select (checked) - brand - High Contrast.chromium.png|69| Changed|
|Table layout table.single select (checked) - neutral - High Contrast.chromium.png|51| Changed|
|Table layout table.single select (checked) - neutral - RTL.chromium.png|2| Changed|
|Table layout table.single select (checked) - neutral.chromium.png|2| Changed|
|Table layout table.single select - High Contrast.chromium.png|25| Changed|
|Table layout table.size - medium - High Contrast.chromium.png|25| Changed|
|Avatar Converged.badgeMask - RTL.normal.chromium.png|0| Added|
|Avatar Converged.badgeMask.normal.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv9"/>